docs/Configuration.md

```diff
@@ -83,6 +91,8 @@ configuration variables
 
 * `SCCACHE_DIR` local on disk artifact cache directory
 * `SCCACHE_CACHE_SIZE` maximum size of the local on disk cache i.e. `2G` - default is 10G
+* `SCCACHE_DIRECT` enable direct mode
+* `SCCACHE_NODIRECT` disable direct mode
```

Comment:
I am personally not a big fan of having two variants of env vars for the same thing.

Reply:
Same answer as above